internal/gitsync/gitsync.go

```diff
@@ -318,3 +325,13 @@ func (a *basicAuth) SetAuth(r *gohttp.Request) {
 		}
 	}
 }
+
+func (s *Synchronizer) updateStartMetrics(startTime time.Time) {
+	metrics.GitSyncCount.Inc()
+	metrics.LastGitSyncStart.WithLabelValues(s.sourceName, s.config.Repo).Set(float64(startTime.Unix()))
```

Comment:
I'm not certain that the `*Start` and `*End` _gauges_ are the way to 🤔 It's a frequent event, and we observe their durations. Why would we _need_ start and end gauges? Perhaps the `*End`  ones are sufficient?